tests/data/fixture_data.py

```diff
@@ -163,50 +164,45 @@
         FIELD_RNA_ID: "rna_1",
         FIELD_LH_SAMPLE_UUID: "8e595a92-6798-4c93-8dc8-44f3ffb8bed3",
         FIELD_DATE_TESTED: "2020-05-10 07:30:00 UTC",
+        FIELD_FILTERED_POSITIVE: False,
     },
-    {  #  positive, with low Ct values
+    {  # positive, with low Ct values
```

Comment:
Are these tests with CT values relevant now? Comment not right at least, as you've removed the CQ values

Reply:
True, will have a look, thanks.